src/ImageSharp/project.json

```diff
@@ -46,7 +46,31 @@
     "System.Runtime.CompilerServices.Unsafe": "4.0.0"
   },
   "frameworks": {
+    "netstandard1.3": {
```

Comment:
We might want to wait a while before we take the RTM version? Been working on it all day on Friday and it was not very friendly. Had a couple of unexpected crashes.